### PR TITLE
saturn.xml: Metadata cleaning

### PR DESCRIPTION
**Pull request by ArcadeShadow**
Renaming Prototype, Alt, Rev, Demo words to lowercase

### CODE REVIEW

File: hash/saturn.xml [hunk 1]
```diff
@@ -39035,7 +39035,7 @@ Olympic Soccer (Fra) T-7904H-09
 
 	<!-- Identifying Sonic 3D (Tech Demo - Oct 08)... -->
 	<software name="sonic3dtd" cloneof="sonic3d" supported="no">
-		<description>Sonic 3D (Tech Demo Oct 08)</description>
+		<description>Sonic 3D (tech demo oct 08)</description>
```
Comment (0kmg) [inline hash/saturn.xml:39038]: It should probably remain `Oct` since months are always capitalized?

File: hash/saturn.xml [hunk 2]
```diff
@@ -18824,7 +18824,7 @@ Olympic Soccer (Fra) T-7904H-09
 	<!-- Identifying Lupin the 3rd Chronicles (Lupin + Fujiko Version) Disc 1 (T-18805G)... -->
 	<!-- Identifying Lupin the 3rd Chronicles (Lupin + Fujiko Version) Disc 2 (T-18805G)... -->
 	<software name="lupincr" supported="no">
-		<description>Lupin the 3rd Chronicles (Lupin + Fujiko Version) (Jpn)</description>
+		<description>Lupin the 3rd Chronicles (Lupin + Fujiko version) (Jpn)</description>
```
Comment (cuavas) [inline hash/saturn.xml:18827]: I believe the “Version” in this case is part of the title, and should be capitalised as such.

File: hash/saturn.xml [hunk 3]
```diff
@@ -30330,7 +30330,7 @@ Olympic Soccer (Fra) T-7904H-09
 	<!-- Identifying 0691-Atlantis-fre-cd1... -->
 	<!-- Identifying 0691-Atlantis-fre-cd2... -->
 	<software name="atlantisf" cloneof="atlantis" supported="no">
-		<description>Atlantis - Secrets d'un monde oublié (Fra)</description>
+		<description>Atlantis - Secrets d'un Monde Oublié (Fra)</description>
```
Comment (cuavas) [inline hash/saturn.xml:30333]: The cover art for this uses a colon to separate the title and subtitle, with a space before the colon, and only capitalises the first letter of the subtitle: https://www.youtube.com/watch?v=EgalDwWbhCI

File: hash/saturn.xml (hunk 1 above)
Comment (cuavas) [inline hash/saturn.xml:39038]: Month names are proper nouns in English and should be capitalised.

**Review by cuavas — CHANGES_REQUESTED** [review]

File: hash/saturn.xml (hunk 1 above)
Comment (ArcadeShadow) [reply hash/saturn.xml]: Corrected

File: hash/saturn.xml (hunk 3 above)
Comment (ArcadeShadow) [reply hash/saturn.xml]: I removed the capitalized letters, keeping only the first subtitle's word.
I kept the dash between the title and subtitle
https://www.amazon.fr/Atlantis-Secret-monde-oubli%C3%A9-Saturn/dp/B01N4J84RK

File: hash/saturn.xml (hunk 2 above)
Comment (ArcadeShadow) [reply hash/saturn.xml]: corrected

File: hash/saturn.xml (hunk 3 above)
Comment (cuavas) [reply hash/saturn.xml]: What are you using from that to justify the use of a dash?  It’s not using punctuation at all there, relying on font size to make the distinction.  In cases like that MAME typically uses a colon anyway.  For example in model3.cpp, the Virtua On 2 games use a colon to separate the title and subtitle even though media just displays the subtitle in a different, smaller font.

Comment (ArcadeShadow) [reply hash/saturn.xml]: I used the dash for no particular reason, I just putted like it was before I changed it at the first place. But it's fine to me to use the colon character. So I changed it again.
But in this case, the parent set "Atlantis - The Lost Tales (Euro)" should have the same treatment, am I correct?

Comment (cuavas) [reply hash/saturn.xml]: Well, it does seem to be conventionally rendered that way, e.g. here: https://en.wikipedia.org/wiki/Atlantis:_The_Lost_Tales

Also, the French title should use French rules for spacing around punctuation, not English rules.  You place a space before a colon (les deux-points) in French, but not in English:
* See the title of the page here: https://fr.wikipedia.org/wiki/Atlantis_:_Secrets_d%27un_monde_oubli%C3%A9
* See French usage examples here: https://study.com/academy/lesson/french-punctuation-marks-rules.html

**Review by cuavas — CHANGES_REQUESTED** [review]
Please don’t force English spacing rules onto French titles.  In French, a space is conventionally placed before a colon (les deux-points).

**Comment by ArcadeShadow** [comment]
Thanks for the spacing rules onto French titles.
I've made the corrections.

**Pull request merged**